lib/auth/auth.go

```diff
@@ -3185,6 +3164,26 @@ func (a *Server) GetDatabase(ctx context.Context, name string) (types.Database,
 
 // ListResources returns paginated resources depending on the resource type..
 func (a *Server) ListResources(ctx context.Context, req proto.ListResourcesRequest) (*types.ListResourcesResponse, error) {
+	// Because WindowsDesktopService does not contain the desktop resources,
+	// this is not implemented at the cache level and requires the workaround
+	// here in order to support KindWindowsDesktop for ListResources.
+	if req.ResourceType == types.KindWindowsDesktop {
```

Comment:
Is testing this new behavior along with ListResources feasible? (In an unit test, not the full integration one you talked about.)

Reply:
ah yes, add to TestListResources so that it's also called directly against the auth server impl (it fails before this change)

Reply:
Nice, thanks!